.github/workflows/ci.yml

```
@@ -435,9 +435,19 @@ jobs:
           if [[ "$PACKAGE" == "server" ]]; then
             pnpm --filter server exec vitest --run --coverage --reporter=default --reporter=github-actions --reporter=junit --outputFile.junit="$JUNIT_OUTPUT"
           else
-            pnpm --filter client exec vitest --run --reporter=default --reporter=junit --outputFile="$JUNIT_OUTPUT"
+            pnpm --filter client exec vitest --run --coverage --reporter=default --reporter=junit --outputFile="$JUNIT_OUTPUT"
           fi
 
+      - name: Upload coverage reports to Codecov
+        if: steps.has-changes.outputs.run == 'true' && secrets.CODECOV_TOKEN != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)